persistence/shared_sql.go

```diff
@@ -14,10 +14,10 @@ import (
 // for the purpose of being explicit: https://github.com/pokt-network/pocket/pull/140#discussion_r939731342
 // TODO Cleanup with #149
 const (
-	UndefinedStakingStatus = 0
-	UnstakingStatus        = 1
-	StakedStatus           = 2
-	UnstakedStatus         = 3
+	UndefinedStakingStatus = int32(0)
```

Comment:
I'm not sure why this change was made within this scope and I don't see it reflected in the Changelog. Can you elaborate here please?

Reply:
The tests were breaking

Reply:
It can be reproduced via `go test -v -count=1 -v ./utility/...` on the main branch

Reply:
![Screen Shot 2022-10-06 at 12 01 35 PM](https://user-images.githubusercontent.com/1892194/194396958-123753be-c1d7-4f80-b3f7-a6ccbbff6e44.png)

Until we get this PR in, `make test_all` does not catch errors

Reply:
That's fine, but can you indicate in the changelog that you fixed broken unit tests? I'll go ahead and approve to not block

Reply:
Done